deploy/manifests/BUILD.bazel

```diff
@@ -39,23 +40,33 @@ VARIANTS = {
     ],
 ) for (name, values) in VARIANTS.items()]
 
+[licensed_file(
+    name = "%s.crds.yaml" % name,
+    src = "%s.crds.unlicensed" % name
+) for (name, values) in VARIANTS.items()]
+
 [genrule(
-    name = name,
+    name = "%s.unlicensed" % name,
     srcs = [
-        "%s.crds" % name,
+        "%s.crds.unlicensed" % name,
```

Comment:
We refer to the unlicensed CRD file here as the overall file will have the license header and we don't want to include it multiple times 😄